### ISSUE TITLE
ngramr error in r: 'subscript out of bounds' or 'NULL' in environment

### ISSUE THREAD

**Issue by jpulido12** [seancarmody/ngramr#39]
Hello, my name is Joemari. I'm submitting this issue to inform you of a bug I think is in the package ngramr. I was using the ngramr package to plot frequencies of words I could potentially use in my independent study as stimuli. However, when I was running the program in r there was an error that popped up: 'subscript out of bounds' or the frequency plots would just read 'NULL' in the environment. I've even copied and pasted the examples you've given in this repository and it still yields the same errors. Here is a screenshot: 
<img width="1440" alt="ngramr_error" src="https://user-images.githubusercontent.com/86123262/194203993-9f94e36a-9464-4648-ad09-5e6a504d4f1e.png">
<img width="1400" alt="error_ngram" src="https://user-images.githubusercontent.com/86123262/194204104-f2eb7353-c07b-47d0-8251-479c8de03ed6.png">

**Comment by seancarmody** [comment]
Hi Joemari. I will have a look and get back to you asap

**Comment by jpulido12** [comment]
Thank you!!

**Comment by seancarmody** [comment]
Google updated their site (again) which broke my code. I have made some modifications which should make it work now. I will complete testing and republish to CRAN. In the meantime you can try out the development version. This requires use of the dev tools package, so install that if you haven't already and then try:
```
library(devtools)
install_github("seancarmody/ngramr", "develop")
library(ngramr)
```
Let me know if that works.

**Comment by seancarmody** [comment]
Hi, the updated version seems to be working fine and I've published it as 1.9.0. So you should be fine now if you simply update the package to the latest version (no need to worry about the development version as described above). Good luck and let me know if you have any problems.

**Comment by jpulido12** [comment]
Hi! I've updated the package to the latest version and it seems to be working perfectly. Thank you so so much for this amazing package!
